TGM/src/main/java/network/warzone/tgm/modules/kit/KitLoaderModule.java

```diff
@@ -12,7 +12,6 @@
 import java.util.ArrayList;
 import java.util.List;
 
-
 public class KitLoaderModule extends MatchModule {
 
     //@Getter private final List<KitNodeParser> parsers = new ArrayList<>();
```

Comment:
You should probably delete this if you're not planning to use it anymore.

Reply:
I meant the line with the `//@Getter`. GitHub is weird.

Reply:
The code was already commented out before I made any changes to the this class. Should I go ahead and just delete the zombie code?

Reply:
Ah okay, never mind then.